cmd/bosun/sched/check.go

```diff
@@ -185,9 +186,63 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 	if checkNotify && s.nc != nil {
 		s.nc <- true
 	}
+	s.CollectStates()
 	s.Save()
 }
 
+// CollectStates sends various state information to bosun with collect.
+func (s *Schedule) CollectStates() {
+	metadata.AddMetricMeta(
+		"bosun.alerts.current_severity", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by current severity.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.last_abnormal_severity", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by last abnormal severity.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.acknowledgement_status", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by acknowledgement status.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.active_status", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by active status.")
+	for _, state := range s.status {
+		if !state.Open {
+			return
+		}
+		// The tagset of the alert is not included because there is no way to
+		// store the string of a group in OpenTSBD in a parsable way. This is
+		// because any delimiter we chose could also be part of a tag key or tag
+		// value.
+		ts := opentsdb.TagSet{"alert": state.Alert}
+		severity := state.Status().String()
+		lastAbnormal := state.AbnormalStatus().String()
+		var err error
+		err = collect.Put("alerts.current_severity", ts.Copy().Merge(opentsdb.TagSet{"severity": severity}), 1.0)
```

Comment:
Why are we using tags to indicate data instead of values? Alert statuses have integer values. This avoids the problem where you're only sending data when something is present instead of sending whether something is present (which causes unknowns). The argument I can think of to use tags, though, is that the alert status can change if we add others, and the tag strings won't. I'm still leaning toward the integer way, though.